backend/src/main/java/com/google/sps/scraping/BBCGoodFoodRecipeScraper.java

```diff
@@ -53,6 +53,12 @@ public static Recipe scrapeRecipe(String url) {
     }
   }
 
+  /*Created this function to output link to the google search */
```

Comment:
The comment should describe what the function does, so `Outputs link to the Custom Search API` or something similar is sufficient. No need to say `created this function`